common/v2/features/SendAssets/components/fields/AssetDropdown.tsx

```diff
@@ -25,7 +25,7 @@ class AssetOption extends React.PureComponent<OptionComponentProps> {
 
 function AssetDropdown({ assets, name, value, onSelect }: Props<Asset>) {
   const filteredAssets: Asset[] = assets.filter(
-    (asset, index) => assets.indexOf(asset) >= index
+    (asset, index) => assets.map(assetObj => assetObj.uuid).indexOf(asset.uuid) >= index
```

Comment:
Note to self:
The extra `map` seems expensive, and I'm wondering if index order is actually what we want to use